README.rst

```diff
@@ -30,21 +30,21 @@ Documentation
 
 Consult `the official Plone documentation <https://docs.plone.org>`_ with information for different audiences.
 
-For trainings `comprehensive Plone training material <https://training.plone.org>`_ is offered.
+For trainings `comprehensive Plone training material <https://training.plone.org>`_ is available.
 
 
 What is Plone?
 --------------
 
 Plone is a ready-to-run content management system, offering a complete set of features needed by a wide variety of organizations.
 
-Plone is secure by architecture and offers fine grained permission control over content and actions.
+Security is built into Plone's architecture from the ground up.  Plone offers fine grained permission control over content and actions.
 
 Plone is easy to set up, extremely flexible,
 and provides you with a system for managing web content that is ideal for project groups, communities, web sites, extranets and intranets.
 
 - *Plone is easy to install.*
-  You can install Plone with a a click and run installer, and have a content management system running on your computer in just a few minutes.
+  You can install Plone with a a click-and-run installer, and have a content management system running on your computer in just a few minutes.
```

Comment:
Is this still true? The Unified-Installer is not that _click-and-run_ as it was in former times with the graphical installer.

Reply:
Yeah, well, to be honest we should say *Plone is possible to install*